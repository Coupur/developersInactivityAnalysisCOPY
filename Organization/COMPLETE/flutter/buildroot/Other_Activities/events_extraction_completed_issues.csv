id
170286177
170531899
172173643
174427291
174636364
176505151
176827341
177868972
177910799
178428943
178430327
178458526
178471702
178484877
178980511
179567613
179636358
179864714
179896209
179919932
179931404
180143763
180701048
180774749
180986702
181295626
184088979
185749143
189270192
189886728
190176466
190194015
190199528
190210069
190445295
190451922
191165348
191166749
192162453
192701238
193169288
193191809
194718401
202280261
202597900
202606233
203309035
203759246
206883874
212266533
212884813
214216260
215579784
215580727
216235943
216244932
216609924
216927654
217371719
220275754
224316970
224573490
225217838
227532425
228506640
233706072
234302178
234311446
234312543
234316829
234369734
234630131
238698161
239313760
239324412
239331713
239597050
240769002
246910575
246914605
247166767
247177530
247249938
247849010
247853022
248119780
248132679
248561661
248566416
248570978
249233174
251344705
251372458
251805327
251806141
252715072
252757612
252769163
254730612
255388541
257532851
258575067
258642762
259953261
261789668
265442421
266692430
267195182
267339040
267864226
270083988
270351714
270698933
272301490
272326135
272400095
273946798
274365440
280657714
285248488
289378858
289781015
290995418
295319978
296851395
296895033
304560968
304562671
304564141
310996669
313069327
314835872
319099989
320113510
321784509
322412037
323387395
323409042
324513985
324523132
324578692
326302348
326709090
327918572
328638603
333812634
335560454
340838893
341145471
341605860
341718735
342121059
342511154
344570277
344584943
348454981
355291017
355804127
356087822
356090205
357305272
357381056
357822024
358576990
358955487
359057359
359109912
359432945
359854267
359966608
361049090
362222993
362778306
363243254
364117140
367355668
368004454
369813776
370316985
370389350
371901591
374564991
374570606
374587976
375132549
375197260
376921791
377113813
381804335
382807368
382889443
382917778
382970277
384050076
386120830
389505830
390069639
390078919
392326058
392399555
392720712
393611620
394508708
397153536
400956350
402879679
403355974
403367462
403370969
404065284
404080053
407471452
407474126
408438121
409568116
410531027
410922750
414401046
418940676
421306885
423975631
424132200
425553820
426138413
426688294
428474505
428508973
428825662
428932439
431179060
433834827
433968567
434687543
436867072
436869559
437900199
438533347
438539001
438540102
438540895
438541892
438542392
439013922
440246786
440262849
441316014
441467721
441530113
443661557
444143211
444172280
444630425
444651967
445526864
449893477
452270771
452673595
454449415
455902106
456463141
458242353
458925019
461224330
463398523
466434685
466549610
466558987
467052061
467139552
467669542
468393157
468417195
468589236
468842179
469412074
469553392
470031168
470046224
470526221
