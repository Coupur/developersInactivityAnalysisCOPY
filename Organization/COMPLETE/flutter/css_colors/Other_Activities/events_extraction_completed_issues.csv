id
354880561
354916581
354917635
