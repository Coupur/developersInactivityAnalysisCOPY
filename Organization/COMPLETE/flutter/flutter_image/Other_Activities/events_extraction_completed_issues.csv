id
242836252
244533272
245833830
249144194
258590243
258600616
261435233
289450147
302158163
307089121
326251758
384016031
439346923
439356806
465599846
