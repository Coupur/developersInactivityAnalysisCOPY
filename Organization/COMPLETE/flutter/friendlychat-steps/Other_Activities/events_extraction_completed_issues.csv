id
153291766
153355520
155325668
155360032
155385572
155994256
158459054
164343904
219053790
219296740
219387468
219977018
220001305
221440206
222520559
222806999
224305161
226050070
228425595
228425883
228878237
229109763
229139103
229434710
230104169
230162742
234397431
240557553
242140866
242212316
242854389
242857099
254115713
265689657
268135735
268139290
268141154
268141927
268149559
268239006
268240554
268600827
269262215
269263733
269264226
269264690
269264815
269264851
269264883
269267722
270842076
272030383
277311365
284335680
299572746
306210953
308125278
308494257
309115948
309830678
311705444
313583405
315159735
317488614
317549007
320343733
320554599
321045456
325653235
330277351
343035544
352549234
358058596
358471371
359257237
360978908
391472803
397239902
430099182
