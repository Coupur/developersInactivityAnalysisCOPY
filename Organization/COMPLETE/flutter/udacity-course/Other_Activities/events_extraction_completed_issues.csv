id
258922806
260058626
260058809
260058829
260058848
260058928
260058977
260059024
260059051
260059070
260697212
261716612
262214105
264012135
264014005
264014485
264014621
264014668
265091161
265091680
265091791
265091907
265611489
268584721
268600199
270131671
274700213
275127986
275128216
276201919
276238489
279533281
279578303
279578373
283960355
285003772
285819928
286435329
286470866
291446386
292173136
293725699
294148900
294162825
294168902
294169637
294171653
294177169
294488936
295358356
295367145
295630902
295687655
296227134
296237046
296512444
296552377
296571775
297631774
297952716
297975969
298433715
298702218
298777759
300352594
300485161
300793639
303314708
303315577
304006051
304008982
304499665
304638985
305005739
305012052
305294212
305333661
305418834
306222341
306222478
306293194
306637343
307303223
308187151
308705970
308767615
310355821
310356410
310359689
310362092
310405281
310562193
311423292
311472574
311714039
311721193
311814311
312039719
312392655
312413852
312693808
312715827
312749118
313516137
318236936
318545382
318548682
320393102
321691949
321707348
322683653
322829985
323813348
323911663
323969739
324033903
324556903
324650803
325550192
328949405
333001474
335104670
335140537
337317198
337672288
337919883
338431036
339987174
340358002
342821200
347749343
351313114
354071857
358371014
363802398
365229872
372942360
373491383
376970541
379118612
381603407
381991492
384229687
384648017
384660353
389236943
389408809
390030253
390234568
391470493
393617855
394522129
394530408
395644790
399823376
404342734
407403966
410538474
413927775
416238872
416455748
438566513
464926395
465359653
