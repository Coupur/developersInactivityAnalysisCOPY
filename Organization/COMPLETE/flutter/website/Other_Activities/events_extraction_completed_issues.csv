id
105714998
105860593
106412621
106414622
106417668
106585602
108501709
108501867
108715043
108970340
109169003
109192930
109196364
109324820
109380895
109510308
109532168
109544191
109872061
110799154
110810671
110854802
110855011
110878443
111054361
111089958
111291528
111757551
111795532
113132739
113977600
114108153
114136600
114144389
114285999
114361482
114388275
114782654
114919829
114925802
114927020
114938955
115117058
115117418
115171924
115184632
115226596
115354383
115398099
115398256
115410663
115502235
115548175
115549524
115550541
115550674
115551088
115551236
115556154
115562207
115563782
115564977
115570683
115576999
115578167
115595550
115605124
115625887
115629031
115630585
115643819
115658346
115708668
115708772
115752141
115909301
115943411
115995057
116000218
116000892
116015337
116471006
116605510
116610222
116621858
116746678
116784931
116954293
117168032
117209652
118060113
118085303
118089071
118090500
118091888
118100799
118225950
118434731
118520851
118520942
118522237
118523049
118926255
119811295
119816586
119823762
121028710
121121944
121125985
121130223
121138794
121139639
121323921
121358445
121359229
121359319
121359378
121505983
121529705
121552212
121641586
122189283
124844628
124859792
125239898
125241614
125242514
125834651
125861583
399594174
125881785
125890570
397155101
126044308
126062659
387708839
126958707
127797334
127810517
127819222
127830303
127840865
127877434
128590137
128614805
128634602
128645873
128648956
128671887
128926014
128987510
129212369
129226362
129253921
129375449
130726359
130821014
131171793
131484290
132495652
132529290
132570089
133171716
133785579
133808772
133809552
133836053
134100238
134101637
134109915
134132788
134139162
134187076
134421079
134693600
134953051
135138998
135592908
135795463
135834121
135885428
136120780
136183893
136183940
136192219
136240435
136241200
136524379
136544536
136767710
137326837
137329138
137653983
137930646
138031617
138645042
138837678
138837883
138912913
139124513
140443014
140768837
140793279
141922917
141985095
141989790
141992192
142458073
142477163
143108593
143658109
143920787
144019718
144085500
144090047
146119822
146415060
146530065
147799999
147826831
148184351
148188602
149567688
149856096
149894202
150146009
150198003
150207989
150986381
151485412
151692727
151694628
152642940
152649107
152686649
152883365
153496130
154561618
154601619
154606232
155131082
155538174
155859167
157263714
157274443
157277876
157279890
157707521
159235727
159249557
159274303
159300000
160958780
161312614
161455925
161469623
164902246
165179116
166661055
168420584
168717966
169246920
171254354
172007847
172013575
172532968
172571144
173015851
173019635
173234585
173880689
174096753
174771223
175933616
176439716
176486277
176491188
177430535
178938662
178974732
179306540
179567679
180100865
180117145
180180651
181762488
181773000
182665737
182667024
182697448
184245841
184249341
184358275
184370355
184374509
184389884
184449020
184582955
185315620
185366149
185600065
186046377
186057022
186342420
186362276
186377080
186425624
186446039
186635575
186642855
186674546
186677282
186679550
186681595
186867833
186882126
187060167
187147519
187175493
187191259
187267131
187521526
187705352
187740494
187768692
187868701
189665825
191052855
191108219
192064780
192075358
192169491
193002570
193633756
193855597
193910075
194706187
194784668
194851853
195032658
197852151
197982023
197992140
198914713
198954740
198959358
199057360
199154910
199195395
199203397
200218747
200369565
200432258
200759321
200765366
200769810
200771876
200776775
200777015
200777292
200777580
201944932
201963906
202468757
202576558
202945503
203031233
203238071
203451032
204185823
204323486
205027115
205739522
205767320
205847885
206053716
206058479
206359939
206580431
206621922
206627608
206642727
206664481
206944913
206964922
207071443
207194905
207363780
207375010
207755342
207834209
207878296
207888526
208286453
208310181
208319499
208357167
208495713
208542790
208579552
208902718
208951256
209394272
209506415
209666860
209788245
209946382
210017082
210160702
210711678
210995934
211252080
211357554
211494958
211585349
211848183
211850194
212045208
212168291
212170808
212820245
212821126
212834331
212835340
212851763
212877598
212892732
212939223
212941674
213122160
213127026
213196658
213221450
213223905
213232351
213291588
213491647
214046195
214072528
214091165
214102560
214166750
214247727
214252311
214278974
214711200
214890019
214899921
215059008
215128733
215513586
215521024
215584640
215621771
215740597
215778290
215784571
215784806
215803288
215806483
215809971
215871257
216146872
216162219
216255013
216271319
216355742
216439568
216505353
216527490
216527923
216557564
216615137
216724139
216935538
216941126
217163604
217198530
217206658
217563866
217635039
217644356
217677849
217686564
217787878
217951320
218369507
218439643
218490330
218825591
219003492
219276034
219683815
219813368
219823058
219840115
220026834
220134869
220271582
220275014
220295391
220545528
221957226
221959318
221960302
221962562
221963329
222424014
222591081
222673391
222726293
222786100
223004228
223014683
223016006
223048158
223423725
223746234
223750652
223861968
223928669
223959939
224129297
224295728
224337000
224618965
224859922
224863525
224903654
225053841
225065793
225075036
225145634
225150752
225924586
225934826
225948250
226066480
226068548
226106490
226109588
226111904
226125365
226331664
226547564
226553445
226658170
226947725
227223845
227393791
227516326
227672571
227682862
227842125
227994345
228217589
228339493
228342738
228519072
228550436
228607893
228828498
228950045
229098979
229162482
229175688
229184613
229451397
229532544
229660728
229756737
229762124
230220775
230541607
230841479
230977009
231089296
231218911
231419223
231502076
231669386
231722551
231759083
231765444
231802221
231968900
232524841
232638535
232691560
232741753
232854109
232962525
233176052
233679407
234140772
234142020
234147729
234382859
234502344
234697911
234714352
235622889
235641290
236034218
236182526
236190131
236485885
236693111
237724530
237748433
237851850
237953160
237967878
238051478
238341327
238704165
239151673
239289602
239300011
239317847
239425490
239446246
239459978
239501703
239503322
239507888
239518895
239521457
240593490
240632156
241330204
242417127
242436105
242522287
243150541
243596517
243728297
243745523
243977693
244640479
244934007
245526308
245688443
245867214
245966621
246460435
246975583
247526579
247547060
249213335
249561474
250035021
250174652
250310216
250626707
251547566
252397553
252405613
252703393
254128639
254890418
255255857
255372275
255404404
255605544
256016072
256342169
256786039
257028689
257458038
257510024
257793633
257833689
257838781
257844633
257887834
257906677
257979640
258103041
258372893
258573861
258576367
258577016
258646936
258787746
259332952
259757442
260015975
260612587
261436364
261738237
262260009
262638049
263876769
263983239
264709259
264782536
265037179
265097272
265098460
265099747
265114580
265938626
266096290
267015991
267036339
267371322
267375522
267606198
267756991
267792761
267830161
267860690
268042478
268223911
268348506
268485089
268572665
268599356
268830221
268833873
268924685
268986928
269454389
270411520
270460108
270461784
270635900
270846447
270936538
271866515
272351801
272490007
273035606
273644430
274524426
274700828
275050991
275070218
275288178
275360881
275906372
275916779
277508733
277534900
278056529
278450346
278646926
279509210
279728765
279754484
279920305
280050046
280059030
280424861
280592795
280624213
281328658
281567749
281567768
281567919
281758943
281919250
282059453
282200443
282206129
282263933
283000884
283412579
284033787
284895110
284989390
285060696
285533959
286024953
286604981
286788371
286857382
289091220
289148321
289381555
289758909
289939543
290396802
290553376
290558825
292612213
292653112
292852320
293306860
294731515
295674745
295795429
295821258
295840357
295933498
296089795
296544430
296588771
296969045
297150655
297152747
297421410
297470217
297750440
297935315
298129923
298342696
298896081
298921738
298976478
299072149
299449009
299532077
299548143
299899187
299914351
299916693
299963477
299977744
300014130
300078063
300314800
300334298
300366010
300422313
300446005
300463306
300476846
300494166
300554164
300556365
300557980
300633939
300644666
300784428
300842150
300843440
300846828
300886869
301108133
301108871
301115567
301291060
301293496
301299832
301300916
301306182
301335433
301449404
301479982
301617428
301626194
301664060
301673132
301705607
301767960
301789009
301844442
301932778
302001602
302021394
302097013
302141586
302179308
302206869
302257376
302269900
302454461
302764569
302814247
303326400
303595089
303708944
303753111
303780314
303786698
303919815
304014090
304028342
304097023
304135483
304137604
304539292
304546910
304659046
304796292
305250816
305331605
305565843
305665327
306130431
306149126
306183298
306280560
306355500
306578501
306721851
306829574
307188225
307191890
307209315
307237102
307447573
307486862
307487009
307615806
308285531
308337225
308667575
308993434
309085721
309093565
309276983
309323961
309459158
309533897
309779962
309843980
310089938
310127757
310266452
310517581
310523269
310576671
397159121
310954488
311004786
311616567
311633112
311653330
311730801
311743210
311967840
311968644
312009192
312171691
312305537
312324163
312484406
312487232
312514419
312603153
312633092
312639991
312658911
312663275
313113454
313125241
313263015
313410755
313460714
313847289
313892666
314185893
314205420
314295049
314312757
314402771
314640556
314680810
314736790
314903069
315075880
315426462
315771499
315791512
315937738
315952026
315953806
316989424
317326998
317728243
317816738
317883516
317888764
317890780
317947353
318362419
318371139
318372166
318374805
318393443
318396275
318409173
318413269
318415501
318416334
318418404
318437582
318486493
318576523
318652037
318839216
318954693
319241914
319286864
319560954
319598741
319775494
319825254
319873960
319903495
320347698
320386732
320452668
320453761
320487432
320865118
320918277
320932152
320981952
321014634
321020804
321023411
321025838
321215978
321332838
321386749
321637626
322069225
322392256
322558188
322568815
322630554
322801764
322924924
322969298
323274384
323321509
323390341
323418214
323510109
323533948
324016759
324746784
325076320
325119240
325122155
325187480
325448808
325462098
325588434
325650776
326329471
326887646
327342755
327495106
327530052
327638594
328099544
328589079
328683621
329149031
329193494
329246712
329631905
329761097
329860596
330118934
330329842
330338190
330938825
331022709
331064195
331332487
332464786
332597925
332929123
332995790
333503093
334123874
334236921
334323297
334388589
334393767
334518100
335017543
335073854
335430981
335842248
335867691
336074661
336159075
336583651
336779285
336781141
336860037
337697710
337796676
337809165
337827261
337886407
338081169
338191346
338654804
338886080
339165536
339914753
340059600
340352848
340834916
341464380
341622118
341682957
342782999
343021768
343266432
343279734
343369681
343401496
343878299
343939813
343961967
344021267
344075182
344176517
344189858
344220599
344231276
344281268
344400809
344442372
344453101
344480747
345896229
346116289
346651924
346878006
347098837
347179913
347211256
347324660
347652758
348120698
348165898
348176359
348280369
348401213
348482534
348530917
348704303
348735520
348769234
348795315
349063750
349201086
349287597
349294745
349313471
349641491
349690826
349781269
350079118
350228977
350248560
350429077
350742811
350921084
350944561
351210048
351302671
351749766
351814476
351814887
351815473
351906146
352008953
352013106
352035782
352099989
352115557
352115677
352115741
352115835
352115899
352115989
352116578
352117590
352136216
352632749
352655090
352889931
352958339
353079208
353371444
353501487
353604531
353793461
354030474
354114444
354454490
354503650
354667207
354678569
354679723
354680407
354687799
354845948
354850335
354935553
355028959
355038511
355041307
355051912
355052329
355082390
355265337
355509636
355682611
355733917
355829755
355991019
356005312
356029245
356044476
356100363
356214750
356349414
356366337
356389577
356677388
356703882
356707135
356715874
356838295
356906029
356950638
356987404
356997760
357043124
357043869
357202096
357382567
357425281
357496676
357499167
357544440
357583303
357608338
357697947
357838147
357844036
357863781
357875513
357894344
358167937
358224059
358224814
358278006
358316987
358329295
358332643
358336804
358351934
358604523
358687260
358749246
358810017
359119027
359121744
359154017
359384366
359461454
359472985
359580473
359615397
359624186
359690194
359784723
359840837
360062218
360209918
360461972
360477080
360534984
360559954
360675941
360676216
361198599
361333791
361505298
361645508
361846310
361996385
362238394
362266747
362364531
362401378
362481661
362520507
362757641
362765630
362766390
362817263
362861945
362918153
363239082
363255403
363274386
363309819
363355950
363368716
363474303
363475986
363488930
363495178
363499849
363516848
363529235
363550254
363664693
363699027
363699549
363774965
363796200
363796683
364075321
364107221
364141372
364143182
364153618
364175466
364175745
364177280
364180730
364186381
364190056
364219229
364236528
364262807
364321371
364339708
364343601
364346768
364408823
364480605
364524482
364682893
364683978
364688531
364803688
364812670
365067046
365071554
365080511
365138790
365264433
365560090
365561153
365572471
365624353
365624467
365630595
365651446
365652595
365694697
365701779
365742033
365845099
365847384
365883087
365886904
365933544
365953828
365980827
365986290
365989644
365995130
366022198
366056269
366070023
366072677
366097147
366106980
366224227
366225547
366298654
366303246
366319633
366624028
366660240
366706194
366812230
366880318
366907587
366979261
367004570
367072680
367152825
367222539
367250210
367260410
367346888
367383990
367389476
367866391
367882345
367886374
367887883
367889308
367893560
367893991
367896652
367922186
367963880
367970847
367976960
367992418
367999773
368065900
368098134
368127634
368132742
368158526
368195882
368215159
368345700
368372085
368409825
368411064
368435277
368446787
368522650
368528742
368552780
368555545
368612073
368681554
368744263
368750947
368762568
368767318
368776947
368779224
368797774
368807592
368808375
368881585
368885727
368908084
368951286
368959870
368969165
368987386
368987783
369114675
369185972
369203836
369230426
369244612
369254078
369255605
369261826
369264284
369286526
369318753
369321755
369335030
369341964
369411547
369487230
369739101
369739307
369739859
369881292
369897460
369993774
369998053
370001319
370030888
370045228
370061648
370061805
370579315
370673651
370677672
370678377
370799390
370799522
370801976
370816883
370817674
370833320
370841414
371027397
371074393
371108965
371181126
371194384
371218032
371218308
371234971
371294185
371534089
371556234
371567211
371599716
371635442
371690883
371700641
371715372
371717334
371726521
371730866
371751003
371813320
371986857
372019628
372033617
372033751
372038854
372052559
372053599
372082280
372116157
372335627
372551787
372596863
373029526
373053127
373061653
373069627
373090246
373097811
373107384
373184135
373210430
373213229
373223708
373229244
373238608
373243734
373243820
373294889
373300244
373471761
373479517
373613068
373622637
373637746
373686206
373702935
373703215
373834255
373915497
373983290
373997353
374001874
374008413
374010585
374011426
374024089
374049981
374056584
374062507
374081909
374082993
374156265
374158670
374343725
374352592
374483578
374494839
374551447
374574767
374587477
374610708
374658735
374664564
374789463
374817231
374954159
375062187
375084111
375088126
375090188
375097393
375112773
375132900
375135263
375139070
375144540
375156909
375159706
375159873
375160028
375168865
375171078
375182278
375214881
375225733
375225948
375229463
375232356
375234173
375234701
375237529
375249654
375262220
375268196
375301567
375305026
375310866
375313742
375565479
375680554
375683016
375696592
375697109
375717517
375729941
399595654
375757579
375781539
375804052
375843048
375846723
375847323
375848766
375849794
375850588
375997676
376004721
376013673
376046321
376133136
376144472
376154104
376157812
376160564
376179900
376180690
376181154
376181537
376182190
376182557
376182642
376183145
376209028
376360544
376472810
376478307
376480901
376484184
376516179
376525974
376531496
376532793
376552855
376566718
376567183
376569815
376571157
376576233
376586616
376594836
376608954
376627692
376880957
376881507
376883631
376893742
376904733
376905481
376906826
376907874
376908256
376913360
376926339
376927402
376928918
376932723
376936194
376945818
376956816
376987328
376992444
377001672
377007737
377008562
377008806
377009803
377009930
377011536
377012922
377020695
377022987
377032974
377054788
377111445
377172272
377174594
377211442
377219550
377234645
377237697
377430422
377456225
377470272
377509137
377515225
377534867
377567798
377844758
377856241
377911628
377939231
378263807
378320380
378375407
378394100
378436679
378521708
378861318
378872108
379000633
379024772
379200989
379255568
379344005
379663994
379820382
379897819
379934734
380063300
380315864
380393893
380433563
380446511
380464825
380470676
380848671
380899764
380905193
380935017
381274426
381313374
381387832
381392711
381402990
381415694
381433410
381700986
381706604
381718778
381794895
381876850
381971596
382010118
382113232
382351498
382508984
382775521
417570919
382913123
382935954
383251363
383268251
383290328
383417184
383987782
384124591
384125175
384126963
384248522
384350665
384445554
384446300
384493447
384520692
384566089
384591765
384762254
384883783
384885888
384888573
384918148
384934966
384975495
384983773
384999237
385013926
385404879
385413145
385418174
385426810
385482061
385791101
385888174
385910126
385930866
385941220
385956009
385958079
385966823
385973234
385977480
385988300
386298664
386302936
386315594
386335381
386340818
386351836
386359871
386380400
386390364
386395028
386396904
386430072
386442706
386453692
386471239
386563919
386575659
386591941
386597126
386610600
386815074
386815687
386826490
386833116
386845513
386892645
386907669
386909122
386909580
386909738
386909965
386910621
386910799
386910952
386911114
386911297
386911566
386911684
386911915
386912131
386912281
386912419
386924626
386939421
386940089
386943029
387156405
387163626
387183721
387185934
387186184
387217150
387242639
387246706
387247024
387247153
387247249
387262352
387275261
387289756
387291217
387325210
387390274
387430343
387438477
387440857
387443025
387450675
387463433
387474443
387477601
387534186
387537249
387538520
387566219
387634125
387675349
387743492
387772160
387786932
387789546
387798197
387821775
387830953
387834555
387838476
387870716
387873702
387876813
387880299
387881467
387882747
387885688
387891981
387894009
387895157
387904370
387910620
387926358
387936197
387938868
387960338
387960406
387967095
387989091
388012470
388044377
388068411
388112078
388114296
388121659
388181082
388189593
388205337
388213162
388214096
388224455
388233504
388242835
388249139
388249371
388301538
388310381
388343732
388343852
388345417
388347033
388360882
388374591
388382496
388395121
388425970
388443031
388531509
388594295
388608786
388609093
388723384
388742603
388806032
388843083
388848682
388851812
388854666
388863268
388878343
388896804
388932313
388933158
388943282
388944766
388962849
388996650
389035282
389036300
389073888
389078079
389099263
389112881
389179712
389189566
389195159
389197579
389201691
389205537
389214395
389214553
389231069
389356369
389359421
389402914
389427899
389439770
389456657
389457010
389457239
389458501
389476395
389544759
389553477
389707163
389707491
389707825
389708050
389708209
389709518
389709804
389709934
389710056
389710150
389710279
389710463
389710650
389710854
389711020
389711218
389711335
389711439
389711549
389711683
389711831
389712659
389712987
389713197
389746653
389754977
389795839
389796610
389814531
389884495
389901212
389956051
389968218
389983017
389986075
389987324
389997831
390009284
390016564
390101018
390245817
390259464
390451808
390465369
390469582
390475726
390479804
390482784
390488899
390489847
390621493
390716846
390735171
390740016
390815949
390846459
390860138
390889775
390924075
390930503
390944398
391101287
391155928
391326537
391326734
391329849
391330740
391344560
391416842
391420974
391423906
391425693
391478083
391514982
391574513
391735509
391735810
391747276
391747834
391748991
391749288
391749819
391750027
391755801
391756570
391757562
391758148
391758861
391900662
391923232
391925230
392128667
392128989
392129521
392132221
392160782
392163808
392168319
392170687
392171254
392173155
392173920
392174474
392178688
392179177
392233725
392233960
392248640
392371090
392381761
392383450
392397873
392612421
392674306
392814055
392889249
392934485
392937704
393158280
393158829
393369881
393462279
393483676
393533728
393559536
393567838
393605016
393612316
393619550
393619900
393710677
393711070
393715484
393790108
394089011
394105726
394246027
394265147
394316446
394366381
394373820
394446019
394452931
394455069
394460247
394573260
394608648
394713674
394886184
394886563
394909018
394963648
394963872
394995110
394995299
395023129
395058539
395089880
395171030
395338501
395373196
395375775
395375919
395397697
395405242
395493246
395572787
395669939
395726824
395755076
395778751
395781309
395781972
395786143
395876613
395876712
395951564
395952593
396068733
396171201
396310894
396341318
396574127
396657680
396672293
396709997
396736180
397019041
397130132
397154016
397155773
397170812
397185549
397377126
397443790
397474964
397563940
397604135
397612012
397623898
397738295
397767336
397863764
397916764
397991889
398022830
398056937
398109232
398476697
398484071
398564043
398616005
399099315
399101117
399135972
399434608
399466553
399504342
399829297
399833319
399840765
399892347
399896792
399907279
399909056
399956816
399965554
399976443
400017515
400042130
400295472
400336592
400350187
400354892
400377191
400392428
400400498
400470316
400507284
400509144
400645801
400900295
400904963
400907822
400913162
400920319
400923695
400944807
400993706
401013705
401333012
401461774
401469858
401484843
401750841
402292694
402332231
402341297
402461739
402483069
402648529
402649833
402709513
402768137
402787562
402788493
402828145
402862925
402935613
402942038
402993162
403014417
403050342
403181239
403189696
403207788
403213858
403214819
403249016
403263644
403284977
403285592
403286325
403286632
403297415
403301567
403351085
403372815
403373565
403382118
403385103
403471340
403576435
403588318
403887574
403896358
403912413
403924776
403947127
403959004
403985461
403989110
403990756
404017685
404049890
404325075
404363962
404379882
404423345
404423540
404473888
404536445
404562415
404573396
404585968
404718169
404877652
404888373
404896812
404957360
404985296
404996472
405045650
405049546
405063329
405063639
405388406
405401685
405439582
405440789
405447827
405451187
405465229
405470943
405499623
405538534
405922631
405965284
406030463
406101665
406103084
406105148
406189735
406201010
406473956
406501778
406514091
406899155
406968122
406974690
406978436
407047668
407260278
407328850
407402018
407421262
407436621
407489836
407502189
407504106
407512925
407893809
407908830
407934627
407963128
408027934
408309724
408458862
408547413
408748973
408757849
408760237
409122296
409150412
409212094
409375253
409498167
409528578
409615953
409643003
409694302
409910485
409923203
410100922
410377805
410639745
410743215
410899899
410908795
410934313
410942787
410959285
410999597
411000098
411002953
411022050
411126990
411317961
411571114
411825476
411868002
412019410
412141442
412144041
412149953
412183813
412207572
412441432
412628038
412683134
412688301
412688611
412694321
412702749
412717760
413072682
413122437
413143253
413144491
413149213
413149404
413153029
413164808
413193618
413208383
413227132
413350773
413396287
413398478
413526713
413527325
413527863
469911421
413545747
413550595
413561756
413562188
413584754
413598592
413599395
413653093
413658481
413702761
413773017
413782557
413799435
413930439
413998402
414137199
414178998
414851348
414907194
415117297
415160209
415477889
415517049
415888868
415898290
415899851
415908637
415982117
416017539
416208907
416211867
416264440
416293370
416305542
416325628
416586671
416766678
416807981
416931575
417129729
417383672
417624226
417638509
417794073
417796119
417914597
417928421
418053302
418055942
418357722
418596832
418659489
418854815
418924395
419002861
419029558
419029909
419038764
419146421
419169004
419285653
419349589
419367174
419490210
419547679
419557371
419607666
419732251
419736931
419856999
419891630
419894586
419932329
419934385
420198602
420206668
420482602
420629679
420679233
420739470
420742307
420743609
420791954
420802359
420843562
420992301
421033905
421194198
421227099
421230792
421241009
421241238
421267490
421335997
421337234
421365395
421718494
421734490
421749683
421805057
421992457
422156841
422387210
422409859
422638059
423029158
423113030
423540873
423748266
424351608
424355420
424360797
424429952
424844451
425201573
425645763
425724145
426021284
426083248
426296466
426711804
426751363
426779972
426881279
427084171
427194776
427287390
427344040
427463536
427680948
427843888
427952265
427953113
427962876
428401203
428463518
428470078
428493221
429119156
429286933
429505925
429537633
429920631
429995053
430161608
430299142
430564032
430577568
430583450
430642819
430922196
431364879
431784293
432186195
432697649
432726578
432882528
432932722
432997147
433136882
433417806
433501166
433586578
434203122
434492047
434525064
434530850
434954205
435226024
435263985
435283360
435332151
435332460
435494065
435813514
436317305
436325261
436429816
436805442
436879024
437141431
437160321
437366757
437888180
438068933
438360620
438564829
438653527
438814493
438939899
439024809
439237553
439308083
439326328
439425738
439462564
439582302
439790696
440211866
440214500
440401472
440819058
440897287
440972246
440979889
441250520
441386292
441419791
441436227
441446587
441448827
441448863
441510172
441511079
441517218
441549579
441837855
441870545
441884312
441962381
442180446
442789730
442791585
442797541
442925761
443100000
443112786
443118627
443140461
443155221
443178901
443181347
443372997
443378178
443616323
443668795
443936328
444168562
444227026
444797664
445660355
445910384
445911934
446025559
446306640
446362932
446657178
446783723
446812596
446837856
446864829
446878673
446892821
446904562
447025960
447089357
447197261
447207918
447227668
447373619
447530391
447611416
447793484
447823609
447866751
447872788
447930604
448009765
448358154
448592987
448637746
449289828
449319995
449467347
449471594
449523205
449808521
449923332
449936466
450039276
450097178
450100291
450114109
450597502
450745433
451129228
451202176
451222896
451226057
451234520
451276485
451283325
451546352
451574547
451812017
451918793
452205082
452471375
452654431
452659592
453139365
453169354
453177160
453249015
453280029
453457476
453623688
453672419
453685127
453715020
453722291
453787677
453865104
453938491
454170195
454765617
455083311
455158081
455392738
455394436
455531731
455596937
455694375
455930438
455938004
456182129
456241028
456292568
456358589
456400008
456555890
456580381
456596917
456654607
456832618
457154350
457373652
457793697
458150664
458239243
458290741
458671714
458749118
459462157
459558725
459576312
459599128
460278371
460283241
460345962
460490291
460500875
460596494
461647408
461687955
461693175
461802722
461815508
462155572
462281545
462522988
462829244
462941781
463114070
463267589
463445763
463455197
463919490
464094043
464098546
464121614
464763516
464950156
465275722
465385140
465386379
465389299
465454501
465494460
466040043
466054429
466060250
466095810
466131880
466170035
466399659
466404213
466563027
466796504
466933796
467024525
467187349
467219620
467735058
467784581
468362338
468375059
468453532
468486888
468879039
469405910
469549301
469578882
469712834
