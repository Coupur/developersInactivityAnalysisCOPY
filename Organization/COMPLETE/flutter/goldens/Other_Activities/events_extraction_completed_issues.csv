id
329244335
329486380
337708018
338050579
339059526
339660551
339956461
339984604
340357731
346745222
350784205
350864876
354824373
357337238
358965100
359433620
359441376
369572341
377019073
377608363
378028342
393597731
397562496
400938076
402019003
402908646
412212270
413590271
431251293
436440109
436904280
436961319
446752240
447219856
450034331
451032238
451720969
451740718
453635613
469485761
